app/experimenter/experiments/forms.py

```diff
@@ -1279,10 +1279,51 @@ class NormandyIdForm(ChangeLogMixin, forms.ModelForm):
     normandy_id = forms.IntegerField(
         label="Recipe ID",
         widget=forms.TextInput(
-            attrs={"class": "form-control", "placeholder": "Recipe ID"}
+            attrs={"class": "form-control", "placeholder": "Main Recipe ID"}
         ),
     )
 
+    other_normandy_ids = forms.CharField(
+        label="Other Recipe IDs",
+        widget=forms.TextInput(
+            attrs={
+                "class": "form-control",
+                "placeholder": "Other Recipe IDs (Optional)",
+            }
+        ),
+        required=False,
+    )
+
+    def clean(self):
+        # we need to check that both keys exist because if the user enters
+        # a noninteger normandy_id, the key is taken out of cleaned_data
+        # and this method will still be run causing a key error
+
+        if {"normandy_id", "other_normandy_ids"} <= set(self.cleaned_data):
+            if (
+                int(self.cleaned_data["normandy_id"])
+                in self.cleaned_data["other_normandy_ids"]
+            ):
+                raise forms.ValidationError(
+                    {"other_normandy_ids": "Duplicate IDs are not accepted."}
+                )
+
+        return self.cleaned_data
+
+    def clean_other_normandy_ids(self):
+        other_ids_list = []
+
+        if self.cleaned_data["other_normandy_ids"]:
+            ids = self.cleaned_data["other_normandy_ids"].split(",")
+            for item in ids:
+                try:
+                    other_ids_list.append(int(item.strip()))
```

Comment:
List comprehensions make everything better:

    try:
        return [int(i.strip()) for i in self.cleaned_data["other_normandy_ids"].split(",")]
    catch ValueError:
        raise forms.ValidationError(.....)